host/sgx/elf.c

```diff
@@ -80,15 +80,13 @@ static bool _is_valid_elf64(const elf64_t* elf)
         return false;
 
     /* Ensure that multiplying header size and num entries won't overflow. */
-    static_assert(
+    OE_STATIC_ASSERT(
         sizeof(uint64_t) >=
-            sizeof(header->e_phentsize) + sizeof(header->e_phnum),
-        "e_phentsize or e_phnum is too large");
+        sizeof(header->e_phentsize) + sizeof(header->e_phnum));
 
-    static_assert(
+    OE_STATIC_ASSERT(
         sizeof(uint64_t) >=
-            sizeof(header->e_shentsize) + sizeof(header->e_shnum),
-        "e_shentsize or e_shnum is too large");
+        sizeof(header->e_shentsize) + sizeof(header->e_shnum));
```

Comment:
Is there a reason you removed the message from the assertion?

Reply:
OE_STATIC_ASSERT doesn't support messages. But the file and line number from the failure should be sufficient to deduce what went wrong. It will also print the contents of the assertion in the  build error.

Reply:
I removed static_assert because Mike mentioned a desire to use C99 and this is the only place where static_assert is used (rather than OE_STATIC_ASSERT). Although technically OE_STATIC_ASSERT doesn't really work on C99 either.